service/history/configs/config.go

```diff
@@ -376,6 +377,7 @@ func NewConfig(
 
 		TaskSchedulerEnableRateLimiter:           dc.GetBoolProperty(dynamicconfig.TaskSchedulerEnableRateLimiter, false),
 		TaskSchedulerEnableRateLimiterShadowMode: dc.GetBoolProperty(dynamicconfig.TaskSchedulerEnableRateLimiterShadowMode, true),
+		TaskSchedulerRateLimiterStartupDelay:     dc.GetDurationProperty(dynamicconfig.TaskSchedulerRateLimiterStartupDelay, 10*time.Second),
```

Comment:
let's use 30s as default, it's safe to use a higher value since the persistence rate limiter is always in place and can protect the system.